Formula/rdkit.rb

```diff
@@ -34,7 +35,13 @@ def install
     python_executable = Formula["python@3.9"].opt_bin/"python3"
     py3ver = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
     py3prefix = Formula["python@3.9"].opt_frameworks/"Python.framework/Versions/#{py3ver}"
+    # py3prefix on Linux is different from one on macOS 
+    on_linux do
+      py3prefix = Formula["python@3.9"].opt_prefix
+    end
     py3include = "#{py3prefix}/include/python#{py3ver}"
+    # Get homebrew numpy location
+    numpy_include = Formula["python@3.9"].opt_lib/"python#{py3ver}/site-packages/numpy/core/include"
```

Comment:
```suggestion
    # Get homebrew numpy location
    numpy_include = Formula["numpy"].opt_lib/"python#{py3ver}/site-packages/numpy/core/include"
```
`numpy` is installed into its own prefix, and not Python's.

Reply:
Thanks, @carlocab. I've corrected it.